expertise/service/expertise.py

```diff
@@ -102,98 +105,33 @@ def _prepare_config(self, request) -> dict:
             config['model_params']['mfr_checkpoint_dir'] = self.mfr_checkpoint_dir
 
         # Create directory and config file
+        token = config.pop('token')
```

Comment:
what does it mean `pop` here?

Reply:
`pop` gets the value of the token and removes the key-value pair from the dict

To also address your Line 114 comment, this is so the token never gets written to disk as part of the config, and the token gets passed as an argument to the Celery task